gammapy/obs/observation.py

```diff
@@ -38,24 +41,28 @@ class ObservationTable(Table):
     """Observation table (a.k.a. run list).
 
     This is an `~astropy.table.Table` sub-class, with a few
-    convenience methods and the following columns:
-
-    * ``OBS_ID``
-    * ``ONTIME``
-    * ``LIVETIME``
-    * ...
-
+    convenience methods. The format of the observation table
+    is described in :ref:`dataformats_observation_lists`.
```

Comment:
This is actually not obvious to me: does this link work in the HTML docs?
(I don't have time to build the docs locally and check myself.)